forge-gui/res/cardsfolder/upcoming/moxite_refinery.txt

```diff
@@ -0,0 +1,8 @@
+Name:Moxite Refinery
+ManaCost:2
+Types:Artifact
+A:AB$ Charm | Cost$ 2 T RemoveAnyCounter<X/Any/SINGLE_Artifact.YouCtrl,Creature.YouCtrl/an artifact or creature> | Choices$ Artifact,Creature | CharmNum$ 1 | SorcerySpeed$ True | AdditionalDescription$ . Activate only as a sorcery.
```

Comment:
sorry but looks like you missed e.g. `Bolrac-Clan Crusher`
imo it might not need any support...?